src/test/HostActivationTests/NativeHosting/HostContext.cs

```diff
@@ -232,6 +232,55 @@ public void RunApp_GetDelegate(string scenario, string checkProperties)
             propertyValidation.ValidateSecondaryContext(result, SharedTestState.ConfigPropertyName, SharedTestState.ConfigPropertyValue);
         }
 
+#if Broken_TestComponentActivationDelegate
+        // This test is broken
+        // Test is written assuming a full runtime, but this test uses mockcoreclr
+        // Even with that there is something else wrong too
+        // For some reason the typeof(HostContext).AssemblyQualifiedName is not being found...
```

Comment:
The failure I experienced here was probably because this file has dependencies on xunit et al. And it probably does not have a deps.json for itself.  So the AssemblyDependencyResolver was unable to find the dependencies necessary to instantiate this class.